src/Uno.UWP/UI/Colors.cs

```diff
@@ -33,149 +33,149 @@ public static Color Parse(string colorCode)
 				}
 				else
 				{
-					uint color = colorCode switch
+					uint color = colorCode.ToLowerInvariant() switch
```

Comment:
I wonder which is faster/better, this or having a series of `if`/`else if`/.. with `StringComparison.OrdinalIgnoreCase`?

Reply:
In general the C# compiler is _smart_ enough to rewrite `switch` statements using `if/else` or using a collection.

In this case there might be a small hit since a new string will likely be allocated (unless it's already all lowercased). However that would not affect (much) the `Colors` properties since they are cached.

I'll try to benchmark this once all other feedback are fixed on the PR.

Reply:
@spouliot Note: newer version of the compiler (which we are not yet using) has introduced a better code gen for switch over strings, see https://github.com/dotnet/roslyn/pull/66081

Consider benchmarking against this version of the compiler (we will move to this version of the compiler eventually)